backend.native/tests/build.gradle

```diff
@@ -3805,6 +3805,7 @@ if (isAppleTarget(project)) {
                     dependsOn ":${targetName}CrossDistRuntime", ':commonDistRuntime', ':distCompiler'
                 }
 
+                extraOpts "-Xembed-bitcode"
```

Comment:
Why is this required?

Reply:
It looks like it should be wether "-Xembed-bitcode" or "-Xembed-bitcode-marker" because swift code compiles with the same options. And dyld doesn't load these frameworks built without one of these options.